### PR TITLE
DDCE-3095: Bootstrap play upgrade

### PR DESCRIPTION
**Pull request by lewey**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,16 +2,16 @@ import sbt._
 
 object AppDependencies {
 
-  val silencerVersion = "1.7.1"
+  val silencerVersion = "1.7.7"
```
Comment (frikit) [inline project/AppDependencies.scala:5]: could be 1.7.9

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.4.9
+sbt.version=1.5.8
```
Comment (frikit) [inline project/build.properties:1]: coulde be 1.6.2

File: project/plugins.sbt [hunk 3]
```diff
@@ -3,16 +3,16 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.16")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.2")
```
Comment (frikit) [inline project/plugins.sbt:10]: could be 2.0.0

**Review by frikit — APPROVED** [review]
LFGTM couple minor points

**Comment by frikit** [comment]
Could you add before coverage and after coverage? for these changes?

**Comment by lewey** [comment]
Coverage was the same before and after:

Coverage was stmt=[94.20%] branch=[86.67%]

**Pull request merged**